Sources/Apollo/GraphQLQueryWatcher.swift

```diff
@@ -70,3 +74,14 @@ public final class GraphQLQueryWatcher<Query: GraphQLQuery>: Cancellable, Apollo
     }
   }
 }
+
+private extension CachePolicy {
+  var alwaysIncludesServerFetch: Bool {
+    switch self {
+    case .fetchIgnoringCacheCompletely, .fetchIgnoringCacheData, .returnCacheDataAndFetch:
+      return true
+    case .returnCacheDataDontFetch, .returnCacheDataElseFetch:
```

Comment:
If you don't mind, can you put each case on its own line to match existing code? If you're tied up I can fix it post-merge.

Reply:
Will do! Thank you for the reactivity!

Reply:
Thank you for giving me something to procrastinate with 😆